osquery/tables/system/windows/registry.h

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Woh, this is a header? Try not to put implementation code into a header. If you want to make methods/structures available to tests, put everything into the `.cpp` (implementation) and include a companion `.h` which `extern`s the structures and declares the methods.